tests/test.sh

```diff
@@ -2206,7 +2206,7 @@ else
 fi
 
 run_ice "$testdir/includes.h.gch" "local" 0 "keepoutput" $TESTCXX -x c++-header -Wall -Werror -c includes.h -o "$testdir"/includes.h.gch
-run_ice "$testdir/includes.o" "remote" 0 $TESTCXX -Wall -Werror -c includes.cpp -include "$testdir"/includes.h -Winvalid-pch -o "$testdir"/includes.o
+run_ice "$testdir/includes.o" "remote" 0 $TESTCXX -Wall -Werror -c includes.cpp -include includes.h -Winvalid-pch -o "$testdir"/includes.o
```

Comment:
Why does this need changing?

Reply:
Original:
`-include` and  `"$testdir"/includes.h` is actually ignored by codes I removed in the 325aee7. So test pass successfully.

Modified:
325aee7 makes `"$testdir"/includes.h` starts actully to work. But there's no file with that name. Only `includes.h` or `"$testdir"/includes.h.gch' are exist.
So I fixed it working as I have thought it is came from mistake.